packages/shared/src/components/modals/DeleteCommentModal.spec.tsx

```diff
@@ -37,13 +37,6 @@ const renderComponent = (
   );
 };
 
-it('should close modal on cancel', async () => {
```

Comment:
Why did you delete this test?

Reply:
This has been deleted since the cancel button is now replaced with the "Markdown supported" link